src/api/telephony/portability/document/telephony-portability-document.v6.service.js

```diff
@@ -0,0 +1,76 @@
+angular.module('ovh-api-services').service('OvhApiTelephonyPortabilityDocumentV6', ($cacheFactory, $resource, $window, $http) => {
+  const cache = $cacheFactory('OvhApiTelephonyPortabilityDocumentV6');
+  const queryCache = $cacheFactory('OvhApiTelephonyPortabilityDocumentV6Query');
+
+  const interceptor = {
+    response(response) {
+      cache.remove(response.config.url);
+      queryCache.removeAll();
+      return response.resource;
+    },
+  };
+
+  const docResource = $resource('/telephony/:billingAccount/portability/:id/document', {
+    billingAccount: '@billingAccount',
+    id: '@id',
+    documentId: '@documentId',
+  }, {
+    query: {
```

Comment:
This is the default behaviour for `$resource.query` if you don't override it there is no need to declare it

Reply:
@marie-j Does $resource.query return an array by default?

Reply:
@Steffy29 Yes it is (https://docs.angularjs.org/api/ngResource/service/$resource#$resource-returns)